samples/java-eventsourced-counter/src/test/java/com/example/actions/CounterJournalToTopicActionTest.java

```diff
@@ -0,0 +1,43 @@
+/* This code was generated by Akka Serverless tooling.
+ * As long as this file exists it will not be re-generated.
+ * You are free to make changes to this file.
+ */
+package com.example.actions;
+// tag::class[]
+import com.akkaserverless.javasdk.testkit.ActionResult;
+import com.example.actions.CounterJournalToTopicAction;
+import com.example.actions.CounterJournalToTopicActionTestKit;
+import com.example.actions.CounterTopicApi;
+import com.example.domain.CounterDomain;
+import com.google.protobuf.Any;
+import com.google.protobuf.Empty;
+import org.junit.Test;
+import static org.junit.Assert.*;
+
+
+public class CounterJournalToTopicActionTest {
+
+  @Test
+  public void increaseTest() {
+    CounterJournalToTopicActionTestKit testKit = CounterJournalToTopicActionTestKit.of(CounterJournalToTopicAction::new); // <1>
+    int valueToincrease = 1;
+    ActionResult<CounterTopicApi.Increased> result = testKit.increase(CounterDomain.ValueIncreased.newBuilder().setValue(valueToincrease).build()); // <2>
+    assertTrue(result.isReply());
+    CounterTopicApi.Increased replyMessage = result.getReply(); // <3>
+    assertEquals(valueToincrease,replyMessage.getValue());
+  }  
+  // end::class[]
+
+  @Test
+  public void decreaseTest() {
+    CounterJournalToTopicActionTestKit testKit = CounterJournalToTopicActionTestKit.of(CounterJournalToTopicAction::new);
+    // ActionResult<com.example.actions.CounterTopicApi.Decreased> result = testKit.decrease(com.example.domain.CounterDomain.ValueDecreased.newBuilder()...build());
+  }
+
+  @Test
+  public void ignoreTest() {
+    CounterJournalToTopicActionTestKit testKit = CounterJournalToTopicActionTestKit.of(CounterJournalToTopicAction::new);
+    // ActionResult<com.google.protobuf.Empty> result = testKit.ignore(com.google.protobuf.Any.newBuilder()...build());
+  }
+
+}
```

Comment:
we can maybe include that last `}` for completeness. As such the code show on screen is compilable.
```suggestion
// tag::class[]
}
// end::class[]
```